.github/workflows/ci.yml

```diff
@@ -82,6 +82,9 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
+      - run: |
+          BUILD_NUMBER=${{ github.sha }}
+          echo "BUILD_NUMBER=${BUILD_NUMBER::7}" >> "$GITHUB_ENV"
```

Comment:
Takes the first 7 characters of `BUILD_NUMBER`. This is not possible to do directly with `${{ github.sha }}`